layouts/members/single.html

```diff
@@ -29,7 +29,7 @@ <h1 class="col-lg-12 fw-bold display-5 mb-4">
                             {{ i18n "The merit of each application will be reviewed on a case-by-case basis by the Board or by a committee appointed for this purpose, according to the" }} <a href="/p/foundation-bylaws/" class="al-laws-link"> <b>{{ i18n "Bylaws" }}</b> </a> {{ i18n "established by the Board." }}
                         </p>
                         <p>
-                            {{ i18n "Membership guidelines are" }} <a href="https://drive.google.com/file/d/1Z73BBJ5_HY6nMUu3DxnPkFrlXYIpFaQ0/view" class="al-guideline-link" target="_blank"> <b>{{ i18n "available here" }}</b> </a>
+                            {{ i18n "Membership guidelines are" }} <a href="bit.ly/almalinuxmembership" class="al-guideline-link" target="_blank"> <b>{{ i18n "available here" }}</b> </a>
```

Comment:
What's the point of this change?

Reply:
It points to a differently formatted version of the file (PDF vs 'published' document) that is easier to consume. If you're referencing the bit.ly URL, it's just because it's easier to remember for me.